package.json

```diff
@@ -45,6 +45,7 @@
         "@mantine/modals": "^7.16",
         "@mantine/notifications": "^7.16",
         "@mantinex/dev-icons": "^1.1.0",
+        "@phosphor-icons/react": "^2.1.7",
```

Comment:
why pull in additional icon package for a `gear`?  We're also using https://tabler.io/icons which has that.  I'd suggest that if we do want to use phosphor, we remove @mantinex/dev-icons and Tabler in favor of it

Reply:
oh yeah, i meant to leave context on this, devika has decided that's the icon package we're going to use going forward (i didn't have a say in this) - we will have to replace and remove all the tabler icons and use phosphor